core/src/main/java/org/eclipse/hono/util/TenantObject.java

```diff
@@ -113,34 +123,16 @@ public TenantObject setEnabled(final boolean flag) {
     }
 
     /**
-     * Gets the subject DN of this tenant's configured trusted
-     * certificate authority.
+     * Sets the trusted anchor for this tenant's Trusted CA using the given public key and subject DN.
      * 
-     * @return The DN or {@code null} if no CA has been set.
-     */
-    @JsonIgnore
-    public X500Principal getTrustedCaSubjectDn() {
-
-        final JsonObject trustedCa = getProperty(TenantConstants.FIELD_PAYLOAD_TRUSTED_CA);
-        if (trustedCa == null) {
-            return null;
-        } else {
-            return Optional.ofNullable((String) getProperty(trustedCa, TenantConstants.FIELD_PAYLOAD_SUBJECT_DN))
-                    .map(dn -> new X500Principal(dn)).orElse(null);
-        }
-    }
-
-    /**
-     * Sets the trusted certificate authority to use for authenticating
-     * devices of this tenant.
+     * @param publicKey The trusted CA's public key.
+     * @param subjectDn The trusted CA's subject DN.
      * 
-     * @param publicKey The CA's public key.
-     * @param subjectDn The CA's subject DN.
      * @return This tenant for command chaining.
      * @throws NullPointerException if any of the parameters is {@code null}.
      */
     @JsonIgnore
-    public TenantObject setTrustAnchor(final PublicKey publicKey, final X500Principal subjectDn) {
+    public TenantObject addTrustAnchor(final PublicKey publicKey, final X500Principal subjectDn) {
```

Comment:
shouldn't it also accept a not-before and a not-after `Instant`?

Reply:
@sophokles73 : recall that you mention this in #1095 
> 1. ...
> 2. extend the data model of the trusted CAs in the Tenant API with (optional) not-before and not-after properties. At the same time, remove support for storing a full X.509 certificate in the cert property.
> 3. ...

So I intend to add this change in a second PR, ideally following that breakdown we agreed on